recipe/meta.yaml

```diff
@@ -7,26 +7,26 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  fn: pca3d_{{ version }}.tar.gz
   url:
     - {{ cran_mirror }}/src/contrib/pca3d_{{ version }}.tar.gz
     - {{ cran_mirror }}/src/contrib/Archive/pca3d/pca3d_{{ version }}.tar.gz
   sha256: c3e95f50a4d1bdc6b48ad18bc496131bd1601ec2502e5f20684ec0239726f5ae
 
 build:
-  number: 1
-  skip: true  # [win32 or osx]
-
+  merge_build_host: True  # [win]
+  number: 2
+  noarch: generic
```

Comment:
Are you sure this package is noarch?

Reply:
Good question. The recipe update was done via the conda_r_skeleton_helper by recreating it. `noacrh: generic` is according to `# NeedsCompilation: no` in the CRAN meta data, see at the end of meta.yaml. Looking at the source tells that it is indeed 100% R only code https://github.com/cran/pca3d

The build fails because of reasons discussed in https://github.com/conda-forge/r-bpca-feedstock/pull/1 ... There is a possible solution by copying over some lines from the r-rgl feedstock... That can be done if we agree to proceed this way.